NOTICE.md

```diff
@@ -20,7 +20,15 @@ This program and the accompanying materials are made available under the terms
 of the Apache License, Version 2.0 which is available at
 <https://www.apache.org/licenses/LICENSE-2.0>.
 
-SPDX-License-Identifier: Apache-2.0
+> SPDX-License-Identifier: Apache-2.0
+
+Some of the files are also made available under the terms of the MIT License
+which is available at <https://opensource.org/licenses/MIT>.
+
+> SPDX-License-Identifier: Apache-2.0 OR MIT
```

Comment:
This section is a bit confusing, do we have files under MIT only or Apache only?

Reply:
No, it is either
* Apache only
* Apache or MIT

I asked Eclipse EMO and they were fine with the wording but I made it clear that the files are dual-licensed.